airbyte-config/init/src/main/java/io/airbyte/config/init/LocalDefinitionsProvider.java

```diff
@@ -28,41 +21,48 @@
 import java.util.stream.Collectors;
 
 /**
- * This provider contains all definitions according to the local yaml files.
+ * This provider contains all definitions according to the local catalog json files.
  */
 final public class LocalDefinitionsProvider implements DefinitionsProvider {
 
-  public static final Class<?> DEFAULT_SEED_DEFINITION_RESOURCE_CLASS = SeedType.class;
-
-  private final static String PROTOCOL_VERSION = "protocol_version";
-  private final static String SPEC = "spec";
+  private static final String LOCAL_CONNECTOR_CATALOG_PATH = CatalogDefinitionsConfig.getLocalConnectorCatalogPath();
```

Comment:
How would we potentially override this on the Cloud side to point to a different catalog? Should we keep a constructor param for overriding the path?

Reply:
I could move to an env variable to mirror the remote catalog override. Thoughts?